src/App.js

```diff
@@ -267,7 +341,7 @@ export const App = () => {
 		api('POST', '/auth', body, {
 			[ContentTypeHeader]: "application/json",
 			[BearerOwnerIdHeader]: walletData.account.address,
-			[BearerLifetime]: 2,
+			[BearerLifetime]: params.objectId ? 24 : 2,
```

Comment:
This limits lifetime to 24h, but how would user know about it or be able to change?

Reply:
<img width="333" alt="image" src="https://github.com/nspcc-dev/panel-fs-neo-org/assets/32885629/863fad7f-e880-4f3e-8ebf-583192cdc928">

Reply:
Now it's not customizable. Should we do it?

Reply:
OK, let's have it this way for now and leave #175 for the future.